selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/miscellaneous/WorkingWithTerminalTest.java

```diff
@@ -384,4 +381,10 @@ private void openMC(String currentLocation) {
     terminal.typeIntoActiveTerminal(Keys.ENTER.toString());
     terminal.waitTextInFirstTerminal("Modify time");
   }
+
+  protected void waitTextInFirstTerminal() {
```

Comment:
If you add `terminal.waitTextInFirstTerminal(String...  expectedText)` method, it will be possible to extract `getExpectedContent()` method instead of `waitTextInFirstTerminal()`:
```
protected String[] getExpectedContent() {
  return new String[] {"che", "a.txt"};
}
```